packages/mui-material-next/src/Divider/Divider.tsx

```diff
@@ -54,48 +51,52 @@ const DividerRoot = styled('div', {
         styles.textAlignLeft,
     ];
   },
-})(
-  ({ theme, ownerState }) => ({
-    margin: 0, // Reset browser default style.
-    flexShrink: 0,
-    borderWidth: 0,
-    borderStyle: 'solid',
-    borderColor: (theme.vars || theme).palette.divider,
-    borderBottomWidth: 'thin',
-    ...(ownerState.absolute && {
-      position: 'absolute',
-      bottom: 0,
-      left: 0,
-      width: '100%',
-    }),
-    ...(ownerState.light && {
-      borderColor: theme.vars
-        ? `rgba(${theme.vars.palette.dividerChannel} / 0.08)`
-        : alpha(theme.palette.divider, 0.08),
-    }),
-    ...(ownerState.variant === 'inset' && {
-      marginLeft: 72,
-    }),
-    ...(ownerState.variant === 'middle' &&
-      ownerState.orientation === 'horizontal' && {
-        marginLeft: theme.spacing(2),
-        marginRight: theme.spacing(2),
+})<{ ownerState: DividerOwnerState }>(
+  ({ theme, ownerState }) => {
+    const { vars: tokens } = theme;
+
+    return {
+      margin: 0, // Reset browser default style.
+      flexShrink: 0,
+      borderWidth: 0,
+      borderStyle: 'solid',
+      borderColor: tokens.sys.color.outlineVariant,
+      borderBottomWidth: '1px',
+      ...(ownerState.absolute && {
+        position: 'absolute',
+        bottom: 0,
+        left: 0,
+        width: '100%',
       }),
-    ...(ownerState.variant === 'middle' &&
-      ownerState.orientation === 'vertical' && {
-        marginTop: theme.spacing(1),
-        marginBottom: theme.spacing(1),
+      ...(ownerState.variant === 'inset' &&
+        ownerState.orientation === 'horizontal' && {
+          marginLeft: '16px',
+        }),
+      ...(ownerState.variant === 'inset' &&
+        ownerState.orientation === 'vertical' && {
+          marginTop: '16px',
+        }),
```

Comment:
Something is not clear to me regarding the inset prop specs from Material You:

They mention `16px` for the inset, but I think that might be an error because from the [What's new section](https://m3.material.io/components/divider/overview#a7783158-c068-4f06-816d-c90922d404ed), looks like inset dividers have way more than 16px margin.

From Figma, the margin also seems to be `16px`. I think we should go with `16px`, but I just wanted to discuss it as it seems weird from Material's specs side.

What do you think @Best-Sardar @zanivan?

Reply:
I think we should keep `16px` as in the MY docs. 

In the example from _What's new section_ (above), for me, it seems that the divider is actually full width on the list content, and the avatar is aside.

![image](https://github.com/mui/material-ui/assets/37222944/711d74a0-bb05-4aa5-a74b-245a2368954d)

In the majority of examples, it seems to respect the inset margin of `16px`:

<img width="250" alt="Screenshot 2023-10-13 at 14 39 15" src="https://github.com/mui/material-ui/assets/37222944/0a0fe787-bc4e-450c-890f-542ded2038f9">

<img width="250" alt="Screenshot 2023-10-13 at 14 39 15" src="https://github.com/mui/material-ui/assets/37222944/bfdd8a11-2dd8-4680-877f-b98ce2fed821">

<img width="250" alt="Screenshot 2023-10-13 at 14 39 15" src="https://github.com/mui/material-ui/assets/37222944/87ba4753-9ac2-41e4-b314-e6bdd444b439">

Reply:
Makes sense, thanks @zanivan 😊